frontend/src/patientApp/PxpApiService.ts

```diff
@@ -33,6 +33,16 @@ interface UpdatePatientData
   };
 }
 
+export type SelfRegistrationData = Omit<
+  UpdatePatientData,
+  "facilityId" | "address"
+> & {
+  registrationLink: string;
+  address: Omit<UpdatePatientData["address"], "zipCode"> & {
```

Comment:
This is a mess because the `updatePatientData["address"]` type is currently incorrect: the `zipCode` property should actually be `postalCode`. Filing a separate bug for this.

Reply:
Sigh yeah. Thanks for catching this